doc/api/repl.md

```diff
@@ -371,8 +376,9 @@ within the action function for commands registered using the
 added: v0.1.91
 -->
 
-* `options` {Object}
-  * `prompt` {String} The input prompt to display. Defaults to `> `.
+* `options` {Object | String}
+  * `prompt` {String} The input prompt to display. Defaults to `> `
+    (with a trailing space).
```

Comment:
The trailing space is not shown in the rendered page, it is wiped out. Is this not somehow confusing?